webapp/pages/profile/_id/_slug.vue

```diff
@@ -64,20 +66,21 @@
               </client-only>
             </ds-flex-item>
           </ds-flex>
-          <ds-space margin="small">
-            <template v-if="!myProfile">
-              <hc-follow-button
-                v-if="!user.isMuted"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T16:12:54Z" title="Wednesday, January 29th 2020, 5:12:54 pm +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -66,16 +68,23 @@
           </ds-flex>
           <ds-space margin="small">
             <template v-if="!myProfile">
+              <base-button
+                v-if="user.isBlocked"
+                @click="unblockUser(user)"
+                class="unblock-user-button"
```


It's probably hard to target the `.base-button` class directly since we don't have a direct (or close) parent with a unique class name? I'd prefer something like `.profile > .base-button` but that would be a bigger refactoring...